src/style.css

```diff
@@ -2,0 +1,68 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
It is recommended to use fallback fonts. In case the main one doesn't work, the browser will use the fallback font. For example, you can specify a system font like Arial or Helvetica.